src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
what is it?

Reply:
This is like a re-export of global types from create-react-app. For example imported images are considered as strings (import icon from './icon.png';  icon: string)

Just generated by create-react-app automatically.